test/metrics/breakdown.test.js

```diff
@@ -144,7 +159,7 @@ test('does not include breakdown when not sampling', t => {
 
     const { metricsets } = data
 
-    assertMetricSet(t, 'transaction', metricsets, {
+    assertMetricSet(t, 'transaction not sampling', metricsets, {
```

Comment:
Previously this test asserted that `transaction.breakdown.count` was reported.  However, with the new behavior the `transaction.breakdown.count` metric is no longer reported _in this specific scenario_ [because its count is zero](https://github.com/elastic/apm-agent-nodejs/blob/28e77afa0ea542094e4b655dea713c6769c63880/lib/instrumentation/transaction.js#L322).